Posts/meeting-23mar- gem updates

```diff
@@ -0,0 +1,376 @@
+---
+title: Raw IRC chat Mar 23rd 2016
+author: avsm (Anil Madhavapeddy)
+abstract: Raw IRC meeting notes on #mirage on Mar23 2016
+---
+
+```
+### Roll Call  
```

Comment:
Looks good -- if you rearrange it so that the Markdown is out the quotes, then it'll render correctly.  That is, put the header with the hashes above, and the IRC pieces within the backquotes.

Also, just overwrite my file and update it rather than creating a new one, and add yourself to the author list (perhaps ask @Engil if Canopy supports multiple authors first!)